packages/taiko-client/driver/txlist_fetcher/blob.go

```diff
@@ -28,23 +26,31 @@ func NewBlobTxListFetcher(l1Beacon *rpc.BeaconClient, ds *rpc.BlobDataSource) *B
 	return &BlobFetcher{l1Beacon, ds}
 }
 
-// Fetch implements the TxListFetcher interface.
-func (d *BlobFetcher) Fetch(
+// FetchOntake implements the TxListFetcher interface.
+func (d *BlobFetcher) FetchOntake(
 	ctx context.Context,
 	_ *types.Transaction,
-	meta metadata.TaikoBlockMetaData,
+	meta metadata.TaikoBlockMetaDataOntake,
 ) ([]byte, error) {
 	if !meta.GetBlobUsed() {
 		return nil, pkg.ErrBlobUsed
```

Comment:
should be pkg.ErrNoBlobUsed

Reply:
Changed: https://github.com/taikoxyz/taiko-mono/pull/18746/commits/e0912fdf83207815d646c59a0e3a01607674d9e4